app-emulation/libspectrum/libspectrum-1.5.0.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools
+
+DESCRIPTION="Spectrum emulation library"
+HOMEPAGE="http://fuse-emulator.sourceforge.net/libspectrum.php"
+SRC_URI="mirror://sourceforge/fuse-emulator/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="audiofile bzip2 gcrypt zlib"
+
+RDEPEND="dev-libs/glib:2
+	audiofile? ( >=media-libs/audiofile-0.3.6 )
+	bzip2? ( >=app-arch/bzip2-1.0 )
+	gcrypt? ( dev-libs/libgcrypt:0 )
+	zlib? ( sys-libs/zlib )"
+DEPEND="${RDEPEND}
+	dev-lang/perl
+	virtual/pkgconfig"
```

Comment:
These are most likely `BDEPEND`. How's `perl` used in these?

Reply:
Both fuse and libspectrum: Perl is used to generate C header and C implementation files during the build. Perl isn't being used at runtime.

After moving `pkgconfig` to `BDEPEND`:

    $ repoman 
    RepoMan scours the neighborhood...
      RDEPEND.suspect               1
       app-emulation/fuse/fuse-1.6.0.ebuild: 'virtual/pkgconfig'

Reply:
Because you're including BDEPEND in RDEPEND?